.github/workflows/eslint.yml

```diff
@@ -0,0 +1,28 @@
+name: ESLint
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "18"
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run ESLint
+        run: npm run lint
+
+      - name: Fix ESLint
```

Comment:
You can remove this.

When PR is created we can just run npm run lint to check for lint errors.

Running lint:fix is not going to fix issues and update PR.

That is something which you have to do.

Reply:
Removed the Command for lint:fix from workflows